common/components/observer-component.ts

```diff
@@ -50,6 +51,24 @@ export class ObserverComponent {
 		this.hooks.push(hook)
 	}
 
+	/** Subscribe to a priority hook with this observer. This hook will be removed when the observer is destroyed.
+	 * Cards and status effects will destroy their own observers, so you as the user do not need to
+	 * worry about this!
+	 * If you are looking for a hook that will be called after the observer is destroyed (the card is removed
+	 * from the board), please use a status effect instead.
+	 */
```

Comment:
Copying the entire comment will almost certainly lead to documentation rot.
```suggestion
	/** Subscribe to a priority hook with this observer. Functions similarly to `subscribe`. */
```